### ISSUE TITLE
Make it faster?

### ISSUE THREAD

**Issue by tjasmin111** [salman0508/Car-Color-Classification-Using-CNN-#1]
Interesting repo. Your current simple CNN takes 129ms to detect color. Do you have any ideas how to make it faster, maybe 10x faster? Any simpler architectures? Like SVMs, Random Forrests, etc?

**Comment by salman0508** [comment]
yes by reducing the size of middle layer of cnn or also some applied technique to reduce image pixels but image retain its information or also try different algo for reduce timing
